apps/community/src/components/about/club/club-card.css.ts

```diff
@@ -0,0 +1,48 @@
+import { themeVars } from '@aics-client/design-system/styles';
+import { style } from '@vanilla-extract/css';
+
+const cardWrapper = style({
+  display: 'flex',
```

Comment:
```suggestion
  display: themeVars.display.flex,
```
토큰화 되어있는 display속성을 적용안하신 이유가 있나요 ?

Reply:
해당 토큰에 대해서 이번에 고민해보게 되었는데요. 사실 `flex`라는 토큰은 `sprinkles`로 `css.ts` 외에서 스타일을 주입하기 위해 선언됐어요. `display`의 속성 값 또한 타입적으로 잘 되어 있기도 하고요. 토큰을 사용하지 않아도 명확하거나, 토큰을 통해서 일괄적으로 변경할 필요가 없는 토큰에 한에서 토큰 없이 사용해도 좋을 거 같아요

관련하여 의견들 부탁드려요 @m2na7 @SunwooJaeho

Reply:
> 토큰을 사용하지 않아도 명확하거나, 토큰을 통해서 일괄적으로 변경할 필요가 없는 토큰에 한에서 토큰 없이 사용해도 좋을 거 같아요
동의합니다 😃